spec/requests/provider_users_spec.rb

```diff
@@ -0,0 +1,46 @@
+require "rails_helper"
```

Comment:
I'm definitely open to a different opinion here, but to me it feels like feature specs would be a more appropriate way of testing these new pages than request specs. We're more interested in "when the user clicks certain things in the browser where do they end up?" than "if an HTTP client makes a particular request, what is in the HTTP response?"

Reply:
to be honest I would appreciate a chat about this next week. I am not sure what really defines whether a test should be in features vs requests.